libraries/AP_DDS/AP_DDS_Service_Table.h

```diff
@@ -14,8 +14,8 @@ constexpr struct AP_DDS_Client::Service_table AP_DDS_Client::services[] = {
     {
         .req_id = to_underlying(ServiceIndex::ARMING_MOTORS),
         .rep_id = to_underlying(ServiceIndex::ARMING_MOTORS),
-        .srv_profile_label = "ArmMotorsService",
+        .srv_profile_label = "",
```

Comment:
Since we are not gonna use the `.srv_profile_label` its better we remove it from the definition in `Service_table` struct

Reply:
Agreed - I didn't see the label used anywhere, but wasn't sure whether it was for future compat. Will remove.